generate.sh

```diff
@@ -27,10 +27,9 @@ usage() {
     echo "  Either --pkg or --pkgCsv must be specified"
     echo ""
     echo "options:"
-    echo "  --pkg <packageName,version[,tfm]>  A package and version, no spaces, separated by comma.  If the optional"
-    echo "                                       tfm is specified, it will be used in the project that restores the"
-    echo "                                       package."
-    echo "                                       Example: System.Text.Json,4.7.0"
+    echo "  --pkg <packageName,version[,tfm]>  A package and version, no spaces, separated by comma.  If optional semicolon-separated"
+    echo "                                       TFMs are specified, they will be used in the project that restores the package."
+    echo "                                       Example: System.Text.Json,4.7.0,nestandard2.0;net461"
```

Comment:
Consider adding multiple examples to make it clear that the tfms are optional.
```suggestion
    echo "                                       Example: System.Collections,4.3.0"
    echo "                                       Example: System.Text.Json,4.7.0,nestandard2.0;net461"
```

Reply:
Good suggestion, thanks. Added